src/main/java/wordle/Game.java

```diff
@@ -0,0 +1,56 @@
+package wordle;
+
+import wordle.domain.*;
+import wordle.view.InputView;
+import wordle.view.OutputView;
+
+public class Game {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final DictionaryReader dictionaryReader;
+    private final WordSelector wordSelector;
+
+    public Game(final InputView inputView,
+                final OutputView outputView,
+                final DictionaryReader dictionaryReader,
+                final WordSelector wordSelector) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.dictionaryReader = dictionaryReader;
+        this.wordSelector = wordSelector;
+    }
+
+    public void start() {
+        final Dictionary dictionary = dictionaryReader.read();
+        final Word word = dictionary.select(wordSelector);
+        final Answer answer = new Answer(word.word());
+        play(dictionary, answer);
+    }
+
+    private void play(final Dictionary dictionary, final Answer answer) {
+        Attempt attempt = new Attempt();
+        Results results = new Results();
+        outputView.welcome(attempt.last());
+        do {
+            final Guess guess = guess(dictionary);
+            final Result result = answer.examine(guess);
+            results = results.add(result);
+            attempt = attempt.next();
+            outputView.showResults(results, attempt);
+        } while (attempt.isRunning() && results.hasNotAnswer());
+    }
+
+    private Guess guess(final Dictionary dictionary) {
+        try {
+            outputView.insertWord();
+            final String word = inputView.inputWord();
+            if (dictionary.isExist(word)) {
+                return new Guess(word);
+            }
+            return guess(dictionary);
+        } catch (final Exception e) {
+            outputView.wrongWord();
+            return guess(dictionary);
+        }
+    }
```

Comment:
isExist()로 있는지 확인을 따로 진행하셨군요! 흐름이 더 자연스러워진 것 같아요 👍 

그런데, 예외 관련 버그가 있어요..! 😢 
`5글자가 아닌 단어`와 `5글자이지만 사전에 없는 단어`의 경우, `if`문에서 걸러지고 바로 guess()로 재귀를 타버려서 의도와는 다르게 예외가 발생하지 않네요..! 
<img width="395" alt="image" src="https://github.com/woowahan-pjs/java-wordle/assets/100072078/2d2faef1-f19f-4279-8324-ea018924cfad">

Reply:
헉 이런 버그가 있었다니, 정확히는 outputView.wrongWord(); 가 호출되지 않는 이슈군요
코드 중복보다는 아래와 같이 예외를 던지는 방향으로 개발해야겠습니다.

```suggestion
    private Guess guess(final Dictionary dictionary) {
        try {
            outputView.insertWord();
            final String word = inputView.inputWord();
            if (dictionary.isExist(word)) {
                return new Guess(word);
            }
            throw new IllegalArgumentException();
        } catch (final Exception e) {
            outputView.wrongWord();
            return guess(dictionary);
        }
    }
```

Reply:
dd70ea8ed926de1dbb874f86ecd2f41bd0541fdf 에 반영했습니다!